test_scripts/Polices/user_consent_of_Policies/ATF_UpdateDeviceList_on_device_connect.lua

```diff
@@ -0,0 +1,67 @@
+---------------------------------------------------------------------------------------------
+-- Description: 
+--     UpdateDeviceList request from SDl to HMI upon new device connection
+--     1. Used preconditions:
+-- 	      Delete files and policy table from previous ignition cycle if any
+--        Close current connection 
+--     2. Performed steps:
+--        Connect device 
+
+-- Requirement summary: 
+--    [UpdateDeviceList] sending to HMI 
+--
+-- Expected result:
+--    SDL sends UpdateDeviceList to HMI right after new device connects over WiFi
+---------------------------------------------------------------------------------------------
+--[[ General Settings for configuration ]]
+Test = require('user_modules/connecttest_resumption')
```

Comment:
@aarotar will connecttest_resumption.lua be commited separately? I don't see such file in user modules and can't run script